packages/forms-web-app/src/controllers/your-planning-appeal/index.js

```diff
@@ -2,11 +2,19 @@ const { VIEW } = require('../../lib/views');
 
 exports.getYourPlanningAppeal = async (req, res, next) => {
   const { appeal, appealLPD } = req.session;
+  const appealEmail = appeal.aboutYouSection.yourDetails.email;
+  const userEmail = req?.user?._json.preferred_username;
```

Comment:
Might be a good idea to transform this into something more readable via the middleware beforehand.